### ISSUE TITLE
First licence request after upgrade to 4.0.2-wvvmp failing with system code 75 error 

### ISSUE THREAD

**Issue by simongregory** [castlabs/electron-releases#32]
When refreshing content with existing licenses on CDM upgrade the first request for an license results in the error:

`License failed InvalidStateError: Rejected with system code (75) {"stack":"Error: Rejected with system code (75)"}`

All subsequent requests work fine (although not necessarily for the same piece of media that fails - but this may be a problem in our app which behaves as though the license is there but won't play the content).

I don't think this is limited to 4.0.2 and affects all 3x and 4x versions as they show the same behaviour as I've downgraded whilst reproducing this.

**Comment by khwaaj** [comment]
I have tested this and can reproduce the exact same behaviour when requesting a new persistent license after upgrading between CDM versions that updates the system id. Also, when trying to load licenses persisted with the previous CDM all attempts yields the same system error code (which is expected since automatic [migration](
https://github.com/castlabs/electron-releases#migrating-from-an-earlier-castlabs-electron-for-content-security-release) between system ids is not supported).

I ran this by my contacts and apparently this is expected on such upgrades. The first attempt to retrieve a new license upgrades the storage for the new system id which results in the error. The suggested solution is to simply retry in such cases and the next attempt will succeed.

I'll see about adding something about this in the README for future reference.
